packages/workspace/src/ui/index.ts

```diff
@@ -1,27 +1,41 @@
 
 export * from "./ui";
-export * from "../common";
+export * from "../common/common";

```

Comment:
Redenumeste common/common.ts in common/index.ts, sa poate fi importat cu ../common